modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * virt/storage/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling volume snapshot boot sources
+
+Enable volume snapshot boot sources by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
```

Comment:
The above is confusing.  We set the parameter in the StorageProfile associated with the storage class that is being used to store operating system base images.